.github/workflows/auto_updates.yml

```diff
@@ -0,0 +1,50 @@
+# Workflow for updating Rust dependencies with Cargo and Javascript dependencies with npm.
```

Comment:
You caught me trying to slip in an extra change. I still like the change because it makes things less Rust/Cargo specific but it will be reverted since there really isn't anything _wrong_ with the way it was.